edit_course/operations/configure.py

```diff
@@ -429,3 +468,54 @@ def configure_content(instance, url):
             category.delete()
 
     return errors
+
+def get_target_category(category, course=None, course_url=None):
+
+    course_name = None
+    instance_name = None
+
+
+    if not category:
+        return None, _("ExerciseCollection object requires collection_category.")
+
+    if (course is None) == (course_url is None):
```

Comment:
`if course or course_url and not (course and course_url):` couple be a bit readable?